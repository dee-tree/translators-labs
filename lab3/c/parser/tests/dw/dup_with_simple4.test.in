dw 2k dup (?, 6), ?, 5