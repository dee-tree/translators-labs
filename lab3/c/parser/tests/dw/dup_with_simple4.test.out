no. of reserved bytes = 8196
